vulns.go

```diff
@@ -81,10 +81,55 @@ func getVulnsWithoutTicket(flags flags, projectID string, maturityFilter []strin
 	}
 
 	j, err := jsn.NewJson(responseAggregatedData)
+	vulnsWithAllPaths := make(map[string]interface{})
+
+	listOfIssues := j.K("issues").Array().Elements()
+	issueType := listOfIssues[0].K("issueType").String().Value
+
+	// IAC issues are of type configuration and are not supported atm
+	if issueType == "configuration" {
+		customDebug.Debug(" *** INFO *** IAC projects are not supported by this tool, skipping this project")
+		return vulnsWithAllPaths
+	}
+
+	// Code issue
+	// the response from aggregated data is empty for code issues
+	if len(listOfIssues) == 0 {
```

Comment:
what happens in the mixed results environments? Or it can't happen? What about after a move to v3 (admittedly might be hard to guess at this point in time)?